src/redux/app/actions.ts

```diff
@@ -25,9 +29,23 @@ export const toggleTestNet = () => {
     };
 };
 
-export const setNetworkTestNetChainId = (blockchain: string, chainId: number) => {
+export const setNetworkTestNetChainId = (blockchain: Blockchain, chainId: number) => {
     return {
         type: APP_SET_NETWORK_TEST_NET_CHAIN_ID,
         data: { blockchain, chainId }
     };
 };
+
+export const toggleNetwork = (blockchain: Blockchain) => {
+    return {
+        type: APP_TOGGLE_NETWORK,
+        data: { blockchain }
+    };
+};
+
+export const sortNetworks = (sortedNetworks: Array<{ blockchain: Blockchain; order: number }>) => {
```

Comment:
`updateBlockchianOrder`

Reply:
i think you can use `Array<Blockchain>`, as it is now i expect values to be:
```ts
[
    {blockchain: 'ZILLIQA', order: 0},
    {blockchain: 'ETHEREUM', order: 1}
]
```
`order` is the same with element index